resources/views/profile/two-factor-authentication-form.blade.php

```diff
@@ -9,7 +9,7 @@
     </form>
 @else
     {{-- Disable 2FA --}}
-    <form method="POST" action="{{ url('user/two-factor-authentication') }}">
+    <form method="POST" action="{{ route('two-factor.enable') }}">
```

Comment:
@steks89 

Let's change this to `route('two-factor.disable')` -- since it's available.

Reply:
you are right 👍